recipes/python-irodsclient/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "python-irodsclient" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "81dd6718722c357b37dfd197f6d7b51f40adc52048dd2adfc9b53846c1f1a39c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - prettytable
```

Comment:
It looks like `prettytable`, `six` and `xmlrunner` are not `build` requirements.